.github/renovate.json

```diff
@@ -38,9 +40,20 @@
       "groupName": ["DevDependencies (non-major)"],
       "automerge": true
     },
+    {
+      "extends": ["packages:test"],
+      "matchUpdateTypes": ["patch", "minor"],
+      "matchCurrentVersion": "!/^0/",
+      "groupName": "Test packages (non-major)",
```

Comment:
```suggestion
      "groupName": "Test packages (minor and patch)",
```

Reply:
same diff, just tried to keep it consistent with all the other rules and naming convention unless you want me to change those too